OpenEdXMobile/res/values/strings.xml

```diff
@@ -232,6 +232,8 @@
     <string name="label_ended">Ended {date}</string>
     <!-- Label for date when course is ending -->
     <string name="label_ending">Ending {date}</string>
+    <!-- Error Message when course dates are not available -->
+    <string name="course_dates_unavailable_message">Course dates are not currently available.</string>
```

Comment:
is it approved by the product?

Reply:
@mumer92 is this string approved by PO ?

Reply:
@omerhabib26 @farhan 
This string was already in the iOS code base when web view was being used, this error was shown if web view was not loaded properly, so now this string is being used in native dates view if api returns error.